### DISCUSSION TITLE
Issue with Recording Settings in Frigate NVR After Restart

### DISCUSSION THREAD

**Discussion by grimmaldus** [blakeblackshear/frigate#12922] [General]
Hi everyone,

I’m experiencing an issue with Frigate NVR where the recording settings are automatically re-enabled after a restart, despite having disabled them previously. I’ve checked my config.yml file and confirmed that the settings are saved correctly. However, it seems like they are not being respected after a restart.

Has anyone else encountered this issue? Are there specific configurations or logs I should look into to resolve this? Any help or suggestions would be greatly appreciated!

Thanks!

**Comment by NickM-27** [comment]
This is working as expected, For now the best way to handle this is to create an automation that sets the values you want when frigate starts

**Reply by grimmaldus** [reply]
Thanks for your suggestion on using automation to manage settings upon startup. I’m curious about the design decision behind making configuration changes via the web UI non-persistent through soft resets.

**Reply by NickM-27** [reply]
it used to work the way you suggested but many users got confused when things would not match the state of the config that they wanted and also it had some bugs due to the way that MQTT works, so it was removed
